rack-ui/pages/load.py

```diff
@@ -153,9 +145,17 @@ def run_unzip(zip_file_contents, turnstile_clicks):
         # generate a file in which to capture the ingestion status
         status_filepath = get_temp_dir_unique("output")
 
+        # set options for the view data dropdown menu:  1) footprint graphs and sometimes 2) copy-to graph
+        sg_link_footprint = semtk3.get_sparqlgraph_url(SPARQLGRAPH_BASE_URL, conn_json_str=manifest.getFootprintConnection())
+        view_graph_children=[dbc.DropdownMenuItem("Target graphs", href=sg_link_footprint, target="_blank")]  # option to view footprint graphs
+        if manifest.getCopyToGraph() != None:
```

Comment:
```suggestion
        if manifest.getCopyToGraph() is not None:
```

[PEP-8](https://peps.python.org/pep-0008/#programming-recommendations) weighs in on this one. I can't explain all the subtleties myself.